app/src/main/java/com/example/tmdb/modules/HomeViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.tmdb.modules
+
+import androidx.lifecycle.ViewModel
+
+public class HomeViewModel(
```

Comment:
You don't need to explicitly specify `public` modifier. In Kotlin, everything is by default `public`.